README.md

```diff
@@ -28,3 +32,11 @@ For examples, see the [examples](./examples) directory in this project's git rep
 pip install -r requirements.txt
 nosetests
 ```
+
+## Motivation
+
+This repo is to address https://github.com/gunthercox/ChatterBot/issues/528
+
+## License
+
+ChatterBo IO Adapters t is licensed under the [BSD 3-clause license](./LICENSE).
```

Comment:
It looks like there is a stray "t" after the word "Adapters".

Reply:
@gunthercox Thanks Master, i'll fix that